package.json

```diff
@@ -54,12 +54,12 @@
         "fetch-mock": "^9.11.0",
         "gamp": "0.2.1",
         "glob": "^8.1.0",
-        "handlebars": "1.3.0",
+        "handlebars": "^4.7.8",
         "interactjs": "1.3.4",
-        "jquery": "1.9.1",
+        "jquery": "^3.7.1",
```

Comment:
**issue:** This is breaking everything in TAO. Please do not apply blindly a version bump without properly migrating the code, especially when breaking changes are involved.

FYI, we already have tasks for taking care of these particular dependencies, as Martin already spotted.